### PR TITLE
Fix some typos in the documentation.

### PR DESCRIPTION
**Pull request by drwells**
I have been collecting these for awhile.

### CODE REVIEW

File: source/grid/manifold_lib.cc [hunk 1]
```diff
@@ -1914,8 +1914,8 @@ ::compute_chart_points (const ArrayView<const Point<spacedim>> &surrounding_poin
         // enable the structdim 2 optimization
         use_structdim_2_guesses = true;
       else if (spacedim == 3)
-        // otherwise these vectors are roughly orthogonal are roughly
-        // orthogonal: enable the structdim 3 optimization if we are in 3D
+        // otherwise these vectors orthogonal are roughly orthogonal: enable
```
Comment (jppelteret) [inline source/grid/manifold_lib.cc]: I think that there's still an extra "orthogonal" in this sentence.

Comment (drwells) [reply source/grid/manifold_lib.cc]: You're right. Fixed.

**Comment by jppelteret** [comment]
Great, thanks! If you've got nothing more you want to add then we can go ahead and merge this. Since its just documentation I don't think we need to wait for the tester.

**Comment by drwells** [comment]
All changes are approved, so lets merge :)

**Pull request merged**